test/functional/tests/io/test_io_engines.py

```diff
@@ -17,13 +18,15 @@
 from test_utils.size import Size, Unit
 
 mount_point = "/mnt/test"
+runtime = timedelta(minutes=15)
 
 
 @pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
 @pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
 @pytest.mark.parametrizex("cache_mode", CacheMode)
 @pytest.mark.parametrizex("filesystem", Filesystem)
-@pytest.mark.parametrizex("io_engine", IoEngine)
+@pytest.mark.parametrizex("io_engine", [IoEngine.sync, IoEngine.libaio, IoEngine.psync,
+                          IoEngine.vsync, IoEngine.pvsync, IoEngine.posixaio, IoEngine.mmap])
```

Comment:
If a set of available io engines is extended (i.e. io_uring), this list will have to be updated as well. How about removing unwanted elements from the list instead of passing all suitable elements?